governance/SIG Network Charter.md

```diff
@@ -1,97 +1,100 @@
 # SIG Network Charter
-
 This charter adheres to the Roles and Organization Management specified in <sig-governance>.
- Team information may be found in the <readme.md>
+Team information may be found in the [readme.md](https://github.com/o3de/sig-network/blob/main/README.md)
 
-## Overview of SIG
+Last Updated: **23rd Nov 2021**
 
-Two concise lines explaining what this SIG does with bullet points of the major responsibilities
+## Overview of SIG
+This SIG is responsible for communication mechanisms in O3DE related to multiplayer networking and cloud networking. The SIG defines and owns
+communication protocols, methods and approaches to support O3DE client to server, as well as interaction points with cloud services.
 
-- Responsibility
+Responsibilities include:
+* Secure network transport including client+server interactions over TCP or UDP sockets.
+* Network state replication.
+* Restful HTTP(s) support.
 
 ## Goals
 
-- Major goals that SIG seeks to generally achieve
+- Major goals that SIG seeks to generally achieve: TBD
 
 ## Scope
 
 - Generalized overall scope of work
 
 ## In scope
 - Define standard for communication negotiation.
-- Define and maintain implementation to enable compression and secure communication between endpoints
-- Define and implement interface to network counters, monitoring, and statistics
-- Define and implement low level instrumentation APIs
+- Define and maintain implementation to enable compression and secure communication between endpoints.
```

Comment:
Nit: May worth adding could services integration to this general overview section as well.

Reply:
I think we agreed the SIG is not responsible for specific cloud services integrations unless SIG agrees to accept to them. Can add note is that SIG is responsible for mechanisms that enable cloud service integrations, but not integrations with specific cloud services.

Reply:
This is already covered by "Define and maintain code to enable cloud data services communication."